src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt

```diff
@@ -37,9 +38,10 @@ import org.sonarsource.sonarlint.core.rpc.protocol.client.fix.FixSuggestionDto
 class ShowFixSuggestion(private val project: Project, private val file: VirtualFile, private val fixSuggestion: FixSuggestionDto) {
 
     fun show() {
+        var hasNavegated = false
```

Comment:
`hasNavigated` but I don't care for a typo, only if you fix the other typo as well.